internal/objects/service/service_test.go

```diff
@@ -140,6 +140,18 @@ func TestDesiredEnvoyService(t *testing.T) {
 		NetworkType: operatorv1alpha1.NodePortServicePublishingType,
 	}
 	cntr := objcontour.New(cfg)
+	https := int32(30443)
```

Comment:
maybe change to nonstandard port numbers for extra validation the port value comes from this setting and not the default